ble_controller/doc/pic/Architecture_With_RTOS.svg

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What's going on in this picture?

"BLE Controller Library" directly talks to "HW peripherals" doesn't it? It doesn't go through the application to do so.

Reply:
It requires that the application forwards IRQs etc. to the library. For example: `void ble_controller_RADIO_IRQHandler(void);`.

Reply:
Right, but the image is implying that "BLE Controller Library" does not directly interface with HW peripherals, this is wrong.

Reply:
Agree with sebastian here.
Add to a jira task for the next release to fix this?

Reply:
The figure has been updated internally and will be added to the next release :+1: